Maddori.Apple/Maddori.Apple/Global/Literal/TextLiteral.swift

```diff
@@ -26,4 +26,11 @@ enum TextLiteral {
     static let setupNicknameViewControllerTitleLabel = "키고에서 사용할 \n닉네임을 입력해주세요"
     static let setupNicknameViewControllerNicknameTextFieldPlaceHolder = "예) 진저, 호야, 성민"
     static let setupNicknameViewControllerDoneButtonTitle = "입력완료"
+    
+    // MARK: - JoinTeamViewController
+    
+    static let joinTeamViewControllerTitleLabel = "진저님 반가워요!\n이제 팀에 합류해주세요"
```

Comment:
p3;

```suggestion
    static let joinTeamViewControllerTitleLabel = "님 반가워요!\n이제 팀에 합류해주세요"
```
이 부분이 변수로 받을 닉네임이 있어서 Literal로 빼기가 조오오오금 애매한 것 같은데...
혹시 `님 반가워요!\n이제 팀에 합류해주세요` 이 부분만 Literal로 설정해두고 
사용하는 부분에서
```swift
final class JoinTeamViewController: BaseTextFieldViewController {
    var name = "진저"
        override var titleText: String {
            get {
                return name + TextLiteral.joinTeamViewControllerTitleLabel
            }
```
요런식으로 쓰는건 어떨까요오???

Reply:
쪼하욥!